src/Kiota.Builder/Plugins/Models/AppManifestModel.cs

```diff
@@ -144,7 +144,8 @@ internal class Icons
 
 internal class CopilotExtensions
 {
-    public IList<Plugin> Plugins { get; set; } = new List<Plugin>();
+    public IList<Plugin> Plugins { get; set; } = [];
+    public IList<DeclarativeCopilot> DeclarativeCopilots { get; set; } = [];
```

Comment:
Should we also add a property with the `JsonExtensionData` attribute to ensure we do not exhibit data loss in the node if the new properties are added that we may not necessarily be interested in?
 
```suggestion
    public IList<DeclarativeCopilot> DeclarativeCopilots { get; set; } = [];
    [JsonExtensionData]
    public Dictionary<string, Object> AdditionalData { get; set; } = new();
```